NYTPhotoViewer/NYTScalingImageView.m

```diff
@@ -145,18 +145,19 @@ - (void)setupImageScrollView {
 - (void)updateZoomScale {
 #ifdef ANIMATED_GIF_SUPPORT
     if (self.imageView.animatedImage || self.imageView.image) {
+        CGSize imageSize = self.imageView.animatedImage ? self.imageView.animatedImage.size : self.imageView.image.size;
```

Comment:
Could this change have been in a separate PR? Not a huge problem, just curious if it's supposed to be all lumped together.

Reply:
yes, definitely could be a separate PR.